SWLOR.Game.Server/Feature/TrapDefinition/PitfallTrap.cs

```diff
@@ -0,0 +1,36 @@
+﻿using SWLOR.Game.Server.Core;
+using SWLOR.Game.Server.Core.NWNX;
+using SWLOR.Game.Server.Core.NWScript.Enum.Creature;
+
+namespace SWLOR.Game.Server.Feature
+{
+    public class PitfallTrap
+    {
+        /// <summary>
+        /// When this trap is triggered, it'll toss the players to the designated waypoint after a delay.
+        /// </summary>
+        [NWNEventHandler("OnTrapTriggered")]
+
```

Comment:
Remove the empty line here. Attributes should always be placed immediately above the method to ensure it's clear what it's associated with.